src/gar.lua

```diff
@@ -156,6 +156,13 @@ function gar.increaseOperatorStake(from, qty)
 end
 
 -- Utility function to calculate withdrawal details and handle balance adjustments
+---@param qty number # The amount of stake to withdraw
+---@param elapsedTime number # The amount of time that has elapsed since the withdrawal started
```

Comment:
Rename this field to include the time unit (i.e. milliseconds)

Reply:
Same for `totalWithdrawalTime`.

Reply:
[8f13aa5](https://github.com/ar-io/ar-io-network-process/pull/161/commits/8f13aa515d4798eeb8fe2920e20814ba6df7b6d9)